tokio-util/Cargo.toml

```diff
@@ -24,8 +24,9 @@ categories = ["asynchronous"]
 default = []
 
 # Shorthand for enabling everything
-full = ["codec", "compat", "io", "time"]
+full = ["codec", "compat", "io", "time", "udp"]
 
+udp = ["tokio/net"]
```

Comment:
Given that `tokio` crate merged `udp` feature into `net` feature, this may be preferable to be a `net` feature.

Reply:
Certainly, I went back and forth a few times on the naming myself. I'm happy to change to `net`